src/test/run-pass/binding/allow_irrefutable_let_patterns.stderr

```diff
@@ -0,0 +1,8 @@
+warning: the feature `irrefutable_let_patterns` has been stable since 1.32.0 and no longer requires an attribute to enable
```

Comment:
I think this should be moved to `src/test/ui/rfc-2086/allow_irrefutable_let_patterns.stderr` or something like that?